zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/initialize/AuthorizationConfigurer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.engine.processing.identity.initialize;
+
+import io.camunda.security.configuration.ConfiguredAuthorization;
+import io.camunda.zeebe.protocol.impl.record.value.authorization.AuthorizationRecord;
+import io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType;
+import io.camunda.zeebe.protocol.record.value.AuthorizationResourceMatcher;
+import io.camunda.zeebe.protocol.record.value.AuthorizationScope;
+import io.camunda.zeebe.util.Either;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AuthorizationConfigurer
+    implements EntityInitializationConfigurer<ConfiguredAuthorization, AuthorizationRecord> {
+
+  @Override
+  public Either<List<String>, AuthorizationRecord> configure(final ConfiguredAuthorization auth) {
+    final List<String> violations = new ArrayList<>();
+
+    if (auth.ownerType().equals(AuthorizationOwnerType.UNSPECIFIED)) {
+      // TODO: Just an example validation for now. Should use the validations currently in REST
+      //   package later on.
+      violations.add("Authorization owner must not be UNSPECIFIED");
```

Comment:
🔧 As an end user I likely won't have used the `UNSPECIFIED` so the error message here doesn't help me to resolve that. Perhaps we could rephrase it to something like "Authorization owner type not found or incorrect"

Reply:
As  I suspected, Spring won't even be able to construct the configuration object.
```
java.lang.IllegalArgumentException: No enum constant io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType.UER
```
Is that something we'd want to explicitly handle and provide a better error message?